notebooks/cards.clj

```diff
@@ -199,13 +201,17 @@
 ;; html
 (v/html [:h1 \"🧨\"])
 "
-        (nextjournal.clerk.parser/parse-clojure-string {:doc? true})
-        (v/with-viewer :clerk/notebook)
-        (v/with-viewers (v/add-viewers [{:name :clerk/result-block
-                                         :transform-fn (v/update-val (comp v/read-string :text))
-                                         :render-fn '(fn [form]
-                                                       (let [data (eval form)]
-                                                         (try
-                                                           (if (nextjournal.clerk.render/valid-react-element? data) data (v/html [v/inspect data]))
-                                                           (catch js/Error e
-                                                             (v/html [:div.red (.-message e)])))))}])))]))
+     doc
+     (nextjournal.clerk.parser/parse-clojure-string {:doc? true} doc)
+     (update doc :blocks (partial map (fn [{:as b :keys [type text]}]
+                                        (cond-> b
+                                          (= :code type)
+                                          (assoc :result
+                                                 {:nextjournal/value
+                                                  (let [val (eval (read-string text))]
+                                                    ;; FIXME: this won't be necessary once we unify v/html in SCI env to be the same as in nextjournal.clerk.viewer
+                                                    ;; v/html is currently html-render for supporting legacy render-fns
```

Comment:
👆 I'd prefer to fix this in a separate commit. It seems to me that legacy render-fn which use `v/html` in their body do make the browser hang if we unify the behaviour of the html viewer with n.clerk.viewer.